1.0/docs/devguide/properties.md

```diff
@@ -370,40 +370,59 @@ Example:
 In addition to properties, observers can also observe [paths to sub-properties](#observing-path-changes),
 [paths with wildcards](#deep-observation), or [array changes](#array-observation).
 
-### Observing path changes {#observing-path-changes}
+### Observing sub-property changes {#observing-path-changes}
 
-You can also observe changes to object sub-properties using the 
-`observers` array, by specifying a full path (`user.manager.name`)
-as a function argument.
+To observe changes in object sub-properties:
 
-Example:
-
-    Polymer({
+1.  Initialize your object to a default value.
```

Comment:
:heavy_check_mark: changed list to unordered
:heavy_check_mark: removed stuff about initializing obj to default val